### PR TITLE
DDCE-3100 Bootstrap dependency upgrades + test coverage bump

### PR DESCRIPTION
**Pull request by Taiwo-AB**
# DDCE-3100

Please include a summary / description of the change and which issue it fixes.  Include any relevant user needs, context or links to other PRs related to this PR (eg. acceptance tests, environment config).

## Checklist PR Raiser
- [ ]  I've updated the README if there are functionality changes
- [x]  I've executed the project script to check Dependencies and update them if they're non-breaking changes
- [x]  I've executed Scalastyle and fixed any warnings that can be resolved quickly
- [ ]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
- [x]  I've included appropriate tests with any code I've added (Unit, Integration), and it meets Coverage
- [ ]  I've squashed my commits - including the JIRA issue number in the commit message
- [ ]  I've contacted a BA to create a task if any of these actions would result in a larger piece of work

Coverage bump:
Statement coverage - 92 -> 95
Branch coverage - 80 -> 88

Removed all Scalastyle warnings

### CODE REVIEW

**Review by frikit — APPROVED** [review]
AT passed ok

File: app/config/FrontendAppConfig.scala [hunk 1]
```diff
@@ -74,8 +74,11 @@ class FrontendAppConfig @Inject()(
   lazy val apiToken: String = loadConfig("auth.api-token")
   lazy val aesKey: String = loadConfig("auth.aes-key")
 
+  // scalastyle:off magic.number
```
Comment (lmugnai) [inline app/config/FrontendAppConfig.scala:77]: Can we remove these comments by creating a val for the numbers? Thanks

File: app/config/FrontendAppConfig.scala [hunk 2]
```diff
@@ -74,8 +74,13 @@ class FrontendAppConfig @Inject()(
   lazy val apiToken: String = loadConfig("auth.api-token")
   lazy val aesKey: String = loadConfig("auth.aes-key")
 
-  lazy val timeOutSeconds: Int = runModeConfiguration.getOptional[Int]("timeoutDialog.timeoutSeconds").getOrElse(780)
-  lazy val timeOutCountDownSeconds: Int = runModeConfiguration.getOptional[Int]("timeoutDialog.time-out-countdown-seconds").getOrElse(120)
+  private val timeOutSecondAlternative = 780
```
Comment (lmugnai) [inline app/config/FrontendAppConfig.scala:77]:
```suggestion
  private val timeOutSecondsAlternative = 780
```

File: app/config/FrontendAppConfig.scala [hunk 3]
```diff
@@ -74,8 +74,13 @@ class FrontendAppConfig @Inject()(
...
+  private val timeOutCountDownSecondsAlternative = 120
+
+  lazy val timeOutSeconds: Int = runModeConfiguration.getOptional[Int]("timeoutDialog.timeoutSeconds").getOrElse(timeOutSecondAlternative)
```
Comment (lmugnai) [inline app/config/FrontendAppConfig.scala:80]:
```suggestion
  lazy val timeOutSeconds: Int = runModeConfiguration.getOptional[Int]("timeoutDialog.timeoutSeconds").getOrElse(timeOutSecondsAlternative)
```

**Review by lmugnai — APPROVED** [review]

**Pull request merged**